aws-push-notifications-pinpoint-common/src/main/java/com/amplifyframework/pushnotifications/pinpoint/PinpointNotificationPayload.kt

```diff
@@ -17,42 +17,29 @@ package com.amplifyframework.pushnotifications.pinpoint
 
 import androidx.annotation.RestrictTo
 import com.amplifyframework.notifications.pushnotifications.NotificationPayload
-import com.amplifyframework.pushnotifications.pinpoint.utils.PushNotificationsConstants
-import com.google.firebase.messaging.RemoteMessage
-import kotlin.collections.HashMap
 
-@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@RestrictTo(RestrictTo.Scope.LIBRARY)
```

Comment:
what package is this now restricting to.

Reply:
This was typo. It should be library group.